Makefile

```
@@ -36,58 +36,64 @@ install: install.buf
 	pub global activate protoc_plugin
 
 .PHONY: install.go
-install.go:
+install.go: ## install go with dependencies
 	cd server && go mod download && grep _ ./cmd/tools/tools.go | cut -d' ' -f2 | sed 's/\r//' | xargs go install && go mod tidy
 
 .PHONY: install.buf
-install.buf:
+install.buf: ## install buf
 	mkdir -p "$(BIN_INSTALL_DIR)"
 	curl -sSL "https://github.com/bufbuild/buf/releases/download/v$(BUF_VERSION)/buf-$(shell uname -s)-$(shell uname -m)" -o "$(BIN_INSTALL_DIR)/buf"
 	chmod +x "$(BIN_INSTALL_DIR)/buf"
 
 .PHONY: uninstall.buf
-uninstall.buf:
+uninstall.buf: ## uninstall buf
 	rm -f $(BIN_INSTALL_DIR)/buf
 
 .PHONY: gen.all
-gen.all:
+gen.all: ## generate all
 	buf generate
 
 .PHONY: test
-test: test.go test.dart
+test: test.go test.dart ## test go and dart files
 
 .PHONY: test.dart
-test.dart: $(DART_MOCK_TARGETS)
+test.dart: $(DART_MOCK_TARGETS) ## test dart files
 	cd client && flutter analyze && flutter test
 
 $(DART_MOCK_TARGETS): $(DART_MOCK_SRCS)
 	cd client && flutter pub run build_runner build
 
 .PHONY: test.dart-e2e
-test.dart-e2e:
+test.dart-e2e: ## e2e test for dart files
 	cd client && flutter drive --target=test_driver/app.dart
 
 .PHONY: format.dart
-format.dart:
+format.dart: ## format dart files
 	cd client && flutter format .
 	make gen.all
 
 .PHONY: test.go
-test.go: lint.go
+test.go: ## test go files
+	$(MAKE) lint.go
```

이거는 왜 dependency 에서 이걸로 변경하셨나요?

```suggestion
test.go: lint.go ## test go files
```

---

아 커밋을 잘못했습니다. 감사합니다.